f5_heat/resources/f5_ltm_pool.py

```diff
@@ -86,6 +86,7 @@ class F5LTMPool(resource.Resource, F5BigIPMixin):
         )
     }
 
+    @f5_common_resources
```

Comment:
One advantage of moving `f5_common_resources` into a mixin is that it eliminates an import statement.
Also it should be renamed to start with `_VERB`.